src/adapters/README.md

```diff
@@ -0,0 +1,430 @@
+	 * @param {function} cb Callback to be invoked upon completion.
+	 */
+	delete = function(params, cb) {
+		let combinedKey = this.getCombinedKey(this, params);
+
+		Logger.system.debug("Mongo.delete for key=" + combinedKey);
+
+
+		fetch(`${this.ROUTES.DELETE}`, {
+			method: "DELETE",
+			body: JSON.stringify({ key: combinedKey }),
+			headers: {
+				'Content-Type': 'application/json'
+			}
+		})
+		.then(data => {
+			cb(null, { status: "success" });
+		})		
+		.catch(err => {
+			Logger.system.error("Mongo.delete key=" + combinedKey + ", Error", err);
+			cb(err, { status: "failed" });
+		});
+	};
+
+	/**
+	 * Get the prefix used to filter keys for particular topics and key prefixes.
+	 *
+	 * @param {object} params
+	 * @param {string} params.topic The topic
+	 * @param {string} params.keyPrefix The key prefix (optional).
+	 * @private
+	 */
+	getKeyPreface = function(params) {
+		const keyPrefix = "keyPrefix" in params ? params.keyPrefix : "";
+		const preface = `${this.getUserPreface()}:${params.topic}:${keyPrefix}`;
+
+		return preface;
+	};
+
+	/**
+	 * Get prefix for all the users stored data.
+	 * @private
+	 */
+	getUserPreface = function() {
+		const preface = `${this.baseName}:${this.userName}`;
+		return preface;
+	};
+
+	/**
+	 * This method should be used very, very judiciously. It's essentially a method designed to wipe the database for a
+	 * particular user.
+	 */
+	clearCache = function(params, cb) {
+		// Implementation here
+	};
+
+	/**
+	 * Wipes the storage container.
+	 * @param {function} cb
+	 */
+	empty = function(cb) {
+		// Implementation here
+	};
+}
+
+let mongoAdapter = new MongoStorageAdapter("MongoStorageAdapter")
+module.exports = mongoAdapter; // Allows us to get access to the uninitialized object
+
+```
+
+
+## Finsemble Setup
+
+1. Open _configs/application/config.json_ and locate the `storage` object: 
+    ```js
+	"storage": {
+            "defaultStorage": "MongoStorageAdapter",
+            "topicToDataStoreAdapters": {
+                "finsemble": "MongoStorageAdapter",
+                "finsemble.workspace": "MongoStorageAdapter",
+                "finsemble.workspace.cache": "IndexedDBAdapter"
+            },
+            "dataStoreAdapters": {
+                "LocalStorageAdapter": "$applicationRoot/adapters/localStorageAdapter.js",
+                "IndexedDBAdapter": "$applicationRoot/adapters/indexedDBAdapter.js",
+                "MongoStorageAdapter": "$applicationRoot/adapters/MongoStorageAdapter.js"
+            }
+        }
+    ```
+
+1. Add a `dataStoreAdapter` as shown above.
+1. Register your new adapter as `defaultStorage` and for the  `finsemble` and `finsemble.workspace` topics. Note that `finsemble.workspace.cache` is a _high frequency_ topic (e.g., it's read and written every time a component is opened, dragged, resized, etc.) and thus should be maintained as a local storage adapter and _not_ saved to an external datastore.
```

Comment:
```suggestion
1. Register your new adapter as `defaultStorage` so that it is used for any unspecified storage topics.
1. Map the  `finsemble` and `finsemble.workspace` topics to your new adapter in `topicToDataStoreAdapters`. Note that `finsemble.workspace.cache` is a _high frequency_ topic (e.g., it's read and written every time a component is opened, dragged, resized, etc.) and thus should be maintained as a local storage adapter and _not_ saved to an external datastore.
```